data/assets/scene/solarsystem/planets/earth/lagrange_points/l2/l2.asset

```diff
@@ -0,0 +1,122 @@
+local transforms = asset.require("./transforms")
+
+
+
+local circle = asset.resource({
+  Name = "Circle",
+  Type = "HttpSynchronization",
+  Identifier = "circle_image",
+  Version = 1
+})
+
+
+local L2Small = {
+  Identifier = "L2Small",
+  Parent = transforms.L2Position.Identifier,
+  Renderable = {
+    Type = "RenderablePlaneImageLocal",
+    Enabled = asset.enabled,
+    Billboard = true,
+    Size = 4000000,
+    Texture = circle .. "circle.png",
+    BlendMode = "Additive"
+  },
+  Tag = { "lagrange_points_earth_l2_small" },
+  GUI = {
+    Name = "L2 Small",
+    Path = "/Solar System/Planets/Earth/Lagrange Points"
+  }
+}
+
+local L2 = {
+  Identifier = "L2",
+  Parent = transforms.L2Position.Identifier,
+  Renderable = {
+    Type = "RenderablePlaneImageLocal",
+    Enabled = asset.enabled,
+    Billboard = true,
+    Size = 70000000,
+    Texture = circle .. "circle.png",
+    BlendMode = "Additive"
+  },
+  Tag = { "lagrange_points_earth", "lagrange_points_earth_l2" },
+  GUI = {
+    Name = "L2",
+    Path = "/Solar System/Planets/Earth/Lagrange Points"
+  }
+}
+
+local L2SmallLabel = {
+  Identifier = "L2SmallLabel",
+  Parent = transforms.L2Position.Identifier,
+  Renderable = {
+    Type = "RenderableLabel",
+    Enabled = asset.enabled,
+    Text = "L2",
+    FontSize = 50.0,
+    Size = 6.0,
+    MinMaxSize = { 1.0, 100.0 },
+    OrientationOption = "Camera View Direction",
+    BlendMode = "Normal",
+    EnableFading = false
+  },
+  Tag = { "lagrange_points_earth_l2_small" },
```

Comment:
Shouldn't the `"lagrange_points_earth"` tag also be included here?

Reply:
🤷 It wasn't in the original asset. I have literally just copied them over, so I have no idea it should be added or not. 

Should it?

Reply:
I would think it should. So I can turn on/off everything related to the Lagrange points in one script.